lib/xapi/problem.rb

```diff
@@ -63,7 +67,12 @@ def md
       path('%s.md' % slug)
     end
 
-    def path(f)
+    def path(file_name)
+      old_path = root_path(file_name)
+      File.exist?(old_path) ? old_path : File.join(meta_dir, file_name)
+    end
```

Comment:
I'd like to see 2 methods here that make it more explicit which the old location and new location are.
(plus the `path` method which combines them.)

Not also that 'old' and 'new' are usually not good names, since in 6 months time even the new will be old and everything gets confusing.